distribution/docs/src/main/resources/content/_running/application-commands.adoc

```diff
@@ -0,0 +1,178 @@
+:title: Application Commands
+:type: subMaintaining
+:status: published
+:parent: Available System Console Commands
+:summary: Application commands available.
+:order: 01
+
+== {title}
+
+Application commands are used from the ${ddf-admin} application to manage applications in the ${branding}.
```

Comment:
Suggestion: `Application commands are installed with the ${ddf-admin} application to manage applications in the ${branding}.`